docs/github.md

```diff
@@ -102,7 +102,7 @@ something like this:
 ```shell script
 $ # Create a new commit on top of the second-to-last commit in `your-feature`,
 $ # as reviews requested a fix there. 
-$ jj new your-feature- 
```

Comment:
I think this was actually intended - the `-` postfix operator resolves to the parent of the commit (similar to git's `~` operator). See https://github.com/martinvonz/jj/blob/main/docs/revsets.md#functions.

Reply:
Gotcha, I hadn't read the comment above that closely enough. Fixed now.